app/src/apps/chorus_dt.py

```diff
@@ -0,0 +1,134 @@
+    """
+
+    # Load chorus dt data based on chosen code_structure
+    # TODO: improve and standardize data import logic
+    chorus_dt_df = ch.get_structure_data(code_structure=None)
+    timeseries_df = chorus_dt_df.groupby(['mission_start_month'], as_index=False)['cumulative_distance'].sum()
+    fig = go.Figure()
+    fig.add_trace(go.Scatter(x=timeseries_df['mission_start_month'], y=timeseries_df['cumulative_distance'].values, \
+                             mode='lines+markers', line=dict(width=3)))
+    fig.update_layout(plot_bgcolor='white', template='plotly_white', margin={'t': 30, 'r': 30, 'l': 30},\
+                      xaxis=xaxis_format
+                      )
+    return fig
+
+select_prestation_type = dcc.Dropdown(
+    id='select-prestation_type',
+    options=[{'label': 'Train', 'value': 'T'}, {'label': 'Avion', 'value': 'A'}]
+)
+
+card_indicateur = dbc.Card(dbc.CardBody([
+    html.H3("Nombre de trajets"),
+    html.H3('2 300')
+]), className='pretty_container')
+
+def make_card_indicateur(title, value):
+    return dbc.Card(dbc.CardBody([
+        html.P(title),
+        html.H3(value)
+    ]), className='pretty_container')
+
+cards = dbc.CardDeck([
+    make_card_indicateur("Nombre de trajets", "2 300"),
+    make_card_indicateur("Emissions (eCO2)", "2M"),
+    make_card_indicateur("Indicateur X", "XX"),
+    make_card_indicateur("Indicateur Y", "YY"),
+])
+
+layout = html.Div([
+    dbc.Row(html.P('', id='values-selected')),
+    # Cards row
+    dbc.Row([
+        dbc.Col([
+                html.B('', id='selected-entity-show'),
+                dbc.Card(dbc.CardBody([
+                    html.H3('Filtres'),
+                    html.Br(),
+                    dbc.FormGroup([
+                        dbc.Label("Type de prestation"),
+                        select_prestation_type
+                    ])
+
+                ]), className='pretty_container'),
+                dbc.Card(dbc.CardBody([
+                    html.H3('Exporter les données'),
+                    html.Br(),
+                    dbc.Button('Export', id='export')
+                ]), className='pretty_container'),
+                dbc.Jumbotron("Explications sur les graphiques et leur fonctionnement..."),
+        ]),
+        dbc.Col([
+            cards,
+            build_figure_container(title='Évolution temporelles des émissions', id='timeseries-chorus-dt', \
+                                   footer='Explications..'),
+        ], width=9)
+    ]),
+    dbc.Row([
+        dbc.Col([
+            build_figure_container(title='Histogramme ', id='hist-by-distance', \
+                                   footer='Explications..'),
+        ], width=8),
+        dbc.Col([
+            build_figure_container(title='Répartition des émissions par type de déplacement', id='donut-by-prestation', \
+                                   footer='Explications..'),
+        ], width=4),
+
+    ])
+], id="div-data-chorus-dt")
+
+@app.callback(
```

Comment:
Maybe we should have some helper somewhere to manage the "selected-entity / children" value and return what we want :
- organization id for a specific dataset
- services id for a specific dataset (we might need to store in selected entities several ids for some organizations (for example for AC))

This function would split the selected-entity using the delimiter ';' and then return the rights ids. Something like this (in an horrible not functionning pseudo code):
``` python
class EntityHandler:
    def get_organization_service_ids(self, selected_entity, dataset):
         orga, services = selected_entity.split(';')
         if dataset == 'chorus-dt':
             orga = self.get_chorus_code(orga)
             services =  [self.get_chorus_code(service) for service in services]
         return organization_code, service_codes
```

Reply:
True - I think this one might require some more in-depth discussion to allow users to:
- Switch between orgs/services in a same view (via filter panel)
- Select multiple orgs/services

Besides, as you are pointing out there should be mapping between chorus/osfi/odrive for orgs and services.

Adding it as an issue.